packages/core/phase2/lib/generateOperations/resultClassHandlers/handleJudgementWithFinalResult.test.ts

```diff
@@ -204,4 +146,14 @@ describe("handleJudgementWithFinalResult", () => {
     expect(exceptions).toHaveLength(0)
     expect(operations).toHaveLength(0)
   })
+
+  it("should return no operations when results are not on PNC, there are unmatched PNC offences, and the offence is not added by the court", () => {
```

Comment:
We also need the same test to `handleAdjournmentWithJudgement.test.ts`.

Reply:
I took the test out as it was raising operations whether those conditions are met or not, so it was perhaps not relevant? Please let me know if my observation is incorrect